sympy/combinatorics/groups.py

```diff
@@ -0,0 +1,157 @@
+class FreeGroup(Group):
+    r"""
+    Called  with  a  positive  integer rank, FreeGroup
+    returns a free group on rank generators. If optional
+    argument name is given then the  generators are printed
+    as name1, name2 etc., that is, each name is the
```

Comment:
Yes, that is one of the API methods to be included(though still not implemented in new PR)